modules/n1ql/pages/n1ql-language-reference/comparisonfun.adoc

```diff
@@ -4,10 +4,12 @@
 
 {description}
 
-GREATEST(expression1, expression2, \...)
+== GREATEST(expression1, expression2, \...)
 
+=== Description
 Largest non-NULL, non-MISSING value if the values are of the same type; otherwise NULL.
 
-LEAST(expression1, expression2, \...)
+== LEAST(expression1, expression2, \...)
 
+=== Description
```

Comment:
Even though it is a well-known function, it would be great if we could add usage example per function